pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -103,7 +103,7 @@ func (a *Bootstrap) Generate(dependencies asset.Parents) error {
 	a.Config.Systemd.Units = append(
 		a.Config.Systemd.Units,
 		igntypes.Unit{Name: "bootkube.service", Contents: content.BootkubeSystemdContents},
-		igntypes.Unit{Name: "tectonic.service", Contents: content.TectonicSystemdContents, Enabled: util.BoolToPtr(true)},
+		igntypes.Unit{Name: "tectonic.service", Contents: content.TectonicSystemdContents},
```

Comment:
Can you explain why you're removing `Enabled` here?  I'm not actually clear on what it's for, but [its not documented][1] and I haven't gone over to read the upstream source yet ;).

[1]: https://godoc.org/github.com/coreos/ignition/config/v2_2/types#Unit

Reply:
There's a note about this in the 0238a2771 commit message now :).